README.md

```diff
@@ -47,13 +47,13 @@ include(/Users/james/ArcGIS_SDKs/Qt100.4/sdk/toolkit/Plugin/ArcGISRuntimeToolkit
 The toolkit repo has a demo project located in the Examples folder. This project is not meant to be a template to base your projects off of, but rather to be a quick and easy way to test and demonstrate the capabilities of various toolkit components. You can run the demo project by doing the following steps:
 
 1. Open `Examples.qmlproject` in Qt Creator
-2. Go to Projects > Run, and add a new variable called QML2_IMPORT_PATH, and set the value equal to the path of ArcGIS Runtime and the Toolkit. For example, `QML2_IMPORT_PATH=/Users/<username>/ArcGIS_SDKs/Qt100.4/sdk/macOS/x64/qml:/Users/<username>/ArcGIS_SDKs/Qt100.4/sdk/toolkit/Import`
-3. Depending on your system, you may additionally need to add the path to the EsriCommonQt library into an environment variable. On Windows, add it to the PATH, on Linux, add it to LD_LIBRARY_PATH, and on macOS, add it to DYLD_LIBRARY_PATH. For example, `PATH=C:\Program Files (x86)\ArcGIS SDKs\Qt100.4\sdk\windows\x64\bin`.
+2. Go to Projects > Run, and add a new variable called QML2_IMPORT_PATH, and set the value equal to the path of ArcGIS Runtime and the Toolkit. For example, `QML2_IMPORT_PATH=/Users/<username>/ArcGIS_SDKs/Qt100.5/sdk/macOS/x64/qml:/Users/<username>/ArcGIS_SDKs/Qt100.5/sdk/toolkit/Import`
+3. Depending on your system, you may additionally need to add the path to the EsriCommonQt library into an environment variable. On Windows, add it to the PATH, on Linux, add it to LD_LIBRARY_PATH, and on macOS, add it to DYLD_LIBRARY_PATH. For example, `PATH=C:\Program Files (x86)\ArcGIS SDKs\Qt100.5\sdk\windows\x64\bin`.
 4. The qmlscene application on Windows and Linux require that the path to the Qt libraries be specified in the PATH and LD_LIBRARY_PATH, respectively. For example, `PATH=C:\Qt5.9.2\msvc2015_64\bin`.
 5. Run the app and select an example from the drop down.
 
 ## Version
-The Toolkit is forwards compatible with the ArcGIS Runtime API. It is not backwards compatible. This means that version 100.0 of the Toolkit can be used with version 100.4 of the Runtime API. However, version 100.4 of the Toolkit cannot be used with version 100.0 of the Runtime API, as this version of the Toolkit may require functionality from a newer version of the Runtime API.
+The Toolkit is forwards compatible with the ArcGIS Runtime API. It is not backwards compatible. This means that version 100.0 of the Toolkit can be used with version 100.5 of the Runtime API. However, version 100.5 of the Toolkit cannot be used with version 100.0 of the Runtime API, as this version of the Toolkit may require functionality from a newer version of the Runtime API.
```

Comment:
We could say 100.4 instead or 100.0? either works but i'm wondering if it's better to denote just the previous version.

Reply:
Yes, good point. Let me update that.